chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResult.java

```diff
@@ -236,6 +245,13 @@
     private CertificationCriterion criterion;
 
     public CertificationResult() {
+        this.testFunctionality = new ArrayList<CertificationResultTestFunctionality>();
+        this.testToolsUsed = new ArrayList<CertificationResultTestTool>();
+        this.testStandards = new ArrayList<CertificationResultTestStandard>();
```

Comment:
We're going to need the Optional Standards in here too, and maybe the SVAP as well, right? I know OS are blocked, but SVAP is in the code, so shouldn't it be here?